report/4.Design/HardwareTest.tex

```diff
@@ -0,0 +1,24 @@
+\section{Hardware testing}
+To ensure the most accurate of the device, it is important to ensure the quality and precision of the chosen hardware to be at its highest.
+Therefore, a test of the motors has been conducted, checking their precision and quality.
+
+\subsection*{Motors}
+To ensure the capabilities of the motors, the tests have to try out combinations of target angles and movement speed, and to what extent precision is reached, considering the braking distance.
+Therefore a setup combining an angle, and a velocity ranging 0-100 is setup as seen in Appendix~\ref{appendix:motortest}.
+
+The terms stable, small correction and huge correction are used to specify the stopping at the angle, and to elaborate on, not only precision, but the amount of correction needed, to stop at the exact angle specified.
+
+The tests' goal is to start at a zero degree angle, speed up to the specified velocity, and stop when reaching the target angle; just in case the motor will go further, it will automatically speed up, in the opposite direction, until reaching its goal, hence the correction.
```

Comment:
"the goal of the tests" synes jeg lyder bedre.

Reply:
"speed up" er vel forkert i dine tests. det er vel "vi kører med hastighed X eller ikke med hastighed X". så "set speed" eller lign